### ISSUE TITLE
ТЗ К КУРСОВОЙ

### ISSUE THREAD

**Issue by shiiishov** [shiiishov/kursovay#1]
1. Общие сведения

1.1 Наименование системы: разработка автоматизированной системы мониторинга и контроля прачечной самообслуживания.

1.2 Наименование организации заказчика, разработчика системы:
Заказчик: КФ МГТУ им. Н. Э. Баумана
Разработчик: Студент КФ МГТУ им. Н. Э. Баумана группы ИУК5-52Б Шишов З.А.

1.3 Плановые сроки начала и окончания работы по созданию системы:
Плановый срок начала работ – 01.09.2023.
Плановый срок окончания работ – 31.12.2023.

2. Назначение и цели создания (развития) системы

2.1. Назначение системы: автоматизация мониторинга и контроля прачечной самообслуживания.

2.2. Цели создания (развития) системы:
- ускорение процесса обслуживания клиентов;
- повышение качества обслуживания клиентов;
- улучшение контроля за использованием оборудования и расходом материалов.

3. Характеристика объектов автоматизации

3.1. Объекты автоматизации:
- прачечное оборудование;
- материалы для прачечной.

3.2. Характеристика объектов автоматизации:
- количество прачечного оборудования - 5 ед.; (3 стиральных и 2 сушильных машины)
- количество материалов для прачечной - 50 ед. (30 гелей и 20 кондиционеров для белья)

4. Требования к системе

4.1. Функциональные требования:
- мониторинг состояния прачечного оборудования;
- контроль использования материалов для прачечной;
- автоматический расчет стоимости услуг;
- формирование отчетов о работе прачечной;
- добавлять/изменять/удалять данные о клиентах.

4.2. Технические требования:
– программный продукт должен хранить всю информацию в базах данных;
– при вводе данных должна осуществляться проверка на корректность введенных данных;
– надежность и целостность системы.

5. Состав и содержание работ по созданию системы

5.1. Состав и содержание работ:
- разработка технического проекта системы;
- разработка программного обеспечения;
- проведение испытаний и наладка системы.

6. Порядок контроля и приемки системы

6.1. Контроль качества работ и приемка системы осуществляется в соответствии с действующими нормативными документами.

6.2. Приемка системы осуществляется после проведения испытаний и наладки системы, а также проверки ее соответствия требованиям, указанным в техническом задании.

7. Требования к составу и содержанию работ по подготовке объекта автоматизации к вводу системы в действие

7.1. Состав и содержание работ по подготовке объекта автоматизации к вводу системы в действие:
- разработка десктопного приложения для автоматизации прачечной самообслуживания;

8. Требования к документированию

8.1. Документация должна соответствовать требованиям действующих нормативных документов.

8.2. Состав документации:
- техническое задание;
- технический проект системы;
- программная документация;

9. Источники разработки

9.1. Нормативные документы:
- ГОСТ 34.602-89 "Техническое задание на создание автоматизированной системы".

**Comment by wldmr-key** [comment]
> Назначение системы: автоматизация мониторинга и контроля прачечной самообслуживания

для кого предназначено?!

> Объекты автоматизации

неверно. Ни оборудование, ни материалы от твоей работы не автоматизируются

> Функциональные требования

явно не все. Нет функций по получению системой данных для выполнения перечисленных функций

> программный продукт должен хранить всю информацию в базах данных;

выброси = это ни о чем

**Comment by shiiishov** [comment]
> > Назначение системы: автоматизация мониторинга и контроля прачечной самообслуживания

> для кого предназначено?!

да, вы правы, нужно было добавить, что данная автоматизация предназначена для владельца бизнесом прачечных самообслуживания.

> > Объекты автоматизации

> неверно. Ни оборудование, ни материалы от твоей работы не автоматизируются

Правильно понимаю? Нужно в этом пункте написать о том, что автоматизация позволяет, например, управлять расписанием, все "бронировки" по времени для стирки будут отображаться у владельца и пользователей, когда они захотят забронировать её. Ещё, например, мониторинг процесса стирки, можно отслеживать процесс стирки и сколько остается времени до завершения стирки. Ещё мониторинг состояния машинки (нужен ли ремонт) и количества средства для стирки (гель для стирки автоматически дозируется, поэтому его отслеживать можно). 

> > Функциональные требования

> явно не все. Нет функций по получению системой данных для выполнения перечисленных функций

мониторинг состояния прачечного оборудования;
автоматический расчет стоимости услуг;
формирование отчетов о работе прачечной;
добавлять/изменять/удалять данные о клиентах;
функции для мониторинга состояния прачечного оборудования;
функции для автоматического расчета стоимости услуг;
функции для формирования отчетов о работе прачечной;
функции для бронирования стиральной машины;
функции для обновления/изменения/удаления данных о клиентах.

**Comment by wldmr-key** [comment]
ну и сделай финальный вариант измененных пунктов

**Comment by shiiishov** [comment]
1. Общие сведения

1.1 Наименование системы: разработка автоматизированной системы мониторинга и контроля для владельца прачечной самообслуживания.

1.2 Наименование организации заказчика, разработчика системы:
Заказчик: КФ МГТУ им. Н. Э. Баумана
Разработчик: Студент КФ МГТУ им. Н. Э. Баумана группы ИУК5-52Б Шишов З.А.

1.3 Плановые сроки начала и окончания работы по созданию системы:
Плановый срок начала работ – 01.09.2023.
Плановый срок окончания работ – 31.12.2023.

2. Назначение и цели создания (развития) системы

2.1. Назначение системы: автоматизация мониторинга и контроля прачечной самообслуживания.

2.2. Цели создания (развития) системы:
- ускорение процесса обслуживания клиентов;
- повышение качества обслуживания клиентов;
- улучшение контроля за использованием оборудования и расходом материалов.

3. Характеристика объектов автоматизации

3.1. Объекты автоматизации:
- бронь на стиральную машину;
- этапы стирки;
- состояние стиральных машин и количества средства для стирки;

3.2. Характеристика объектов автоматизации:
- количество прачечного оборудования - 5 ед.; (3 стиральных и 2 сушильных машины)
- количество материалов для прачечной - 50 ед. (30 гелей и 20 кондиционеров для белья)

4. Требования к системе

4.1. Функциональные требования:
- мониторинг состояния прачечного оборудования;
- автоматический расчет стоимости услуг;
- формирование отчетов о работе прачечной;
- добавлять/изменять/удалять данные о клиентах;
- функции для мониторинга состояния прачечного оборудования;
- функции для автоматического расчета стоимости услуг;
- функции для формирования отчетов о работе прачечной;
- функции для бронирования стиральной машины;
- функции для обновления/изменения/удаления данных о клиентах.

4.2. Технические требования:
- при вводе данных должна осуществляться проверка на корректность введенных данных;
- надежность и целостность системы.

5. Состав и содержание работ по созданию системы

5.1. Состав и содержание работ:
- разработка технического проекта системы;
- разработка программного обеспечения;
- проведение испытаний и наладка системы.

6. Порядок контроля и приемки системы

6.1. Контроль качества работ и приемка системы осуществляется в соответствии с действующими нормативными документами.

6.2. Приемка системы осуществляется после проведения испытаний и наладки системы, а также проверки ее соответствия требованиям, указанным в техническом задании.

7. Требования к составу и содержанию работ по подготовке объекта автоматизации к вводу системы в действие

7.1. Состав и содержание работ по подготовке объекта автоматизации к вводу системы в действие:
- разработка десктопного приложения для автоматизации прачечной самообслуживания;

8. Требования к документированию

8.1. Документация должна соответствовать требованиям действующих нормативных документов.

8.2. Состав документации:
- техническое задание;
- технический проект системы;
- программная документация;

9. Источники разработки

9.1. Нормативные документы:
- ГОСТ 34.602-89 "Техническое задание на создание автоматизированной системы".

**Comment by wldmr-key** [comment]
> количество прачечного оборудования - 5 ед.; (3 стиральных и 2 сушильных машины)
количество материалов для прачечной - 50 ед. (30 гелей и 20 кондиционеров для белья)

А. если будет 29 гелей - не взлетит? ))

в остальном - приемлемо

**Comment by shiiishov** [comment]
> А. если будет 29 гелей - не взлетит? ))

Могу тогда просто убрать уточнение? И уже это тз распечатывать?

**Comment by wldmr-key** [comment]
разумеется
